docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -420,6 +422,18 @@ table description supplier are:
 * There is no need to define tables manually.
 * Some Protobuf specific types like ``oneof`` are supported and mapped to JSON.
 
+When using Protobuf decoder with the Confluent table description supplier, the
+following additional steps must be taken:
+
+- Copy the JARs ``kafka-protobuf-provider`` and ``kafka-protobuf-types`` for
+  Confluent version 7.3.1 to the Kafka connector plugin directory on all nodes in
```

Comment:
what directory is this? /plugin/kafka

Reply:
And maybe we need a section in the kubnertes page on how to add that file with helm (if that is even possible at the moment with the trino chart)

Reply:
@hashhar is it possible to add the jar files with Helm? Per Manfred's comment above.

Reply:
Maybe just keep the mention generic that it needs to go into the plugin/kafka folder of the installation directory of Trino. And that folder varies between tarball, rpm, and container.

Reply:
Helm is not unique. You add JARs like how you add manually. You place them in a directory and mount that to the container and copy those. However the recommended practice with Docker is to create your own containers and morever this page is not a deployment specific page. We don't want to have instructions for Helm/k8s, OpenShift, Rancher and so on - it's up to whoever is deploying to know how to "Copy a file".

Reply:
> And that folder varies between tarball, rpm, and container.

If you say `plugin/kafka` then that folder doesn't change ever - only it's parent location might change - and that's why we don't use absolute paths in docs.